src/main/java/com/google/sps/servlets/BusinessServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.FileReader;
+import com.opencsv.*;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/business")
+public class BusinessServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String bus= csvToJson("/home/skolb/software-product-sprint/team13/src/main/webapp/businesses.csv");
+      response.setContentType("application/json;");
+      response.getWriter().println(bus);
+  }
+
+private String csvToJson(String file) 
+{ //return json of all rows
+    List<String[]> json= new ArrayList<String[]>();
+	try { 
+
+		// Create an object of filereader 
+		// class with CSV file as a parameter. 
+		FileReader filereader = new FileReader(file); 
+		// create csvReader object passing 
+		// file reader as a parameter 
+		CSVReader csvReader = new CSVReader(filereader); 
+		String[] nextRecord; 
+
```

Comment:
You don't need to do this in this PR, but it would be nice to read this into an Object

Class Business {
  boolean online;
  String name;
  etc..
}

Reply:
I'll try to implement this in the next PR!